code/modules/jobs/job_types/wasteland.dm

```diff
@@ -276,8 +288,8 @@ Preacher
 /*
 Punished Raider
 */
-/*
-/datum/job/f13Punraider
+
+/datum/job/f13punraider
```

Comment:
set positions to 0 if we are going to uncomment this - make it so only admins can activate it